ghc-lib-gen/src/Main.hs

```diff
@@ -51,6 +56,6 @@ ghclibgen (GhclibgenOpts root target ghcFlavor cppOpts) =
         -- 'hadrian/stack.yaml'.
         generatePrerequisites ghcFlavor
         -- Renamings come after 'generatePrerequisites':
+        applyPatchDerivedConstants ghcFlavor -- Needs DerivedConstants.h
```

Comment:
out of curiosity, did it make a difference to do this step earlier?

Reply:
no, not significant i think, i probably just moved it while i was working stuff out. key point is it must follow `generatePrequisites` is all

Reply:
also, if you don't do a `git update` after renaming it (which is what happens in `applyPatchDerivedConstants` iirc), that step will fail for the lack of it

Reply:
or something 😂